testsuite/smoke-test/src/network.rs

```diff
@@ -149,10 +149,10 @@ async fn test_file_discovery() {
                 network.discovery_method = DiscoveryMethod::None;
                 network.discovery_methods = vec![
                     DiscoveryMethod::Onchain,
-                    DiscoveryMethod::File(
-                        (*discovery_file_for_closure2).as_ref().to_path_buf(),
-                        Duration::from_millis(100),
-                    ),
+                    DiscoveryMethod::File(FileDiscovery {
+                        path: discovery_file_for_closure2.path().to_path_buf(),
+                        interval_secs: 1,
```

Comment:
every one second?

Reply:
It's for the test idk

Reply:
I don't want it to be too long or the test will take too long